app/views/ProfileView/index.js

```diff
@@ -181,7 +187,27 @@ class ProfileView extends React.Component {
 
 		// Username
 		if (user.username !== username) {
-			params.username = username;
+			const regEx = new RegExp(`^${ UTF8_Names_Validation }$`);
+			if (regEx.test(username)) {
+				params.username = username;
+			} else {
+				this.setState({
+					saving: false,
+					currentPassword: null
+				});
+				Alert.alert(
```

Comment:
Use `showErrorAlert` instead `Alert` to show error alerts.

Reply:
Ok sure